src/main/java/com/google/api/codegen/config/MethodConfig.java

```diff
@@ -334,10 +329,14 @@ public boolean hasRequestObjectMethod() {
   }
 
   /**
-   * Returns the Interface that should be used for the GRPC call in place of the interface
-   * in which this method appears.
+   * Returns the Interface that should be used for the GRPC call in place of the interface in which
+   * this method appears.
    */
   public String getRerouteToGrpcInterface() {
     return rerouteToGrpcInterface;
   }
+
+  static private boolean isStreamingMethod(Method method) {
```

Comment:
It was only used within this class but I found there are other places can reuse this method. Now I make it public. Done.